src/main/steps/applicant2/finalising-your-application/content.ts

```diff
@@ -0,0 +1,14 @@
+import { TranslationFn } from '../../../app/controller/GetController';
+import { FormContent } from '../../../app/form/Form';
+import {
+  form as applicant1Form,
+  generateContent as applicant1GenerateContent,
+} from '../../applicant1/finalising-your-application/content';
+
+export const form: FormContent = {
+  ...applicant1Form,
```

Comment:
are you sure it uses the same fields as app1?

Reply:
You are correct its a different field, modified the form to include the 'doesApplicant2WantToApplyForFinalOrder' field.